api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -195,11 +195,36 @@ message SatelliteStreamRequest {
     // Status: ALPHA This API is under development and may not work correctly or be changed in backwards
     //         incompatible ways in the future.
     GroundStationConfigurationRequest ground_station_configuration_request = 7;
+
+    // An ack message to respond to each ReceiveTelemetryResponse when `enable_flow_control` is true.
+    //
+    // Status: ALPHA This API is under development and may not work correctly or be changed in backwards
+    //         incompatible ways in the future.
+    ReceiveTelemetryAck telemetry_received_ack = 9;
   }
 
   // The `Framing` types to accept, for satellites that have been configured for multiple framings
   // (e.g., IQ + AX25). If empty, all framings produced by the satellite will be returned.
   repeated Framing accepted_framing = 4;
+
+  // Optional. The `Telemetry.message_ack_id` of the most recently received telemetry. This may be specified
+  // in attempt to resume streaming after a connection or application error. If the specified,
+  // server will attempt to resume transfer assuming the provided value is that of the most recent Telemetry
+  // received & processed by the client.
```

Comment:
What happens if an invalid ID is provided?

Reply:
added this to comments

```
If not specified (or if provided value is invalid), server will resume streaming from the most recent telemetry message sent by the server which may or may not have been received by the client.
```

Reply:
Thanks, makes sense.